### PR TITLE
PLAT-1772: Fix for initials not displayed correctly

### PR DESCRIPTION
**Pull request by mihaelpalfi**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-1772)

- Tried all the different regex expressions (also suggested ones) and could not make it work. :(
- Added override option for initials to all needed components.

#### Description
As per JIRA ticket

#### ChangeLOG

[Changelog](https://zen-ui.zengrc.com/?path=/docs/changelog--page)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

File: src/components/zen-avatar-details/zen-avatar-details.tsx [hunk 1]
```diff
@@ -31,19 +31,22 @@ export class ZenAvatarDetails {
 
   @Element() host: HTMLZenAvatarDetailsElement;
 
-  /** Different variants  */
+  /** Set fifferent variants  */
```
Comment (homer0) [inline src/components/zen-avatar-details/zen-avatar-details.tsx:34]:
```suggestion
  /** Set the style variant  */
```

First I wanted to change the `fifferent` :P, but then I realized that the implementation doesn't set "variants", it chooses a single variant from a list.

**Review by homer0 — APPROVED** [review]
image

🤘

File: src/components/zen-avatar-details/zen-avatar-details.tsx (hunk 1 above)
Comment (mihaelpalfi) [reply src/components/zen-avatar-details/zen-avatar-details.tsx]: Good catch. And i like this fix suggestions!

**Pull request merged**